Makefile

```diff
@@ -11,6 +11,7 @@ MGMT_COMMON_DIR := $(TOP_DIR)/sonic-mgmt-common
 BUILD_DIR := build/bin
 export CVL_SCHEMA_PATH := $(MGMT_COMMON_DIR)/build/cvl/schema
 export GOBIN := $(abspath $(BUILD_DIR))
+export PATH := $(PATH):$(GOBIN):$(shell dirname $(GO))
```

Comment:
why we need this line?

Reply:
It is to resolve gocov not found issue, for go bin and go command not in the PATH.
gocov is in $(GOBIN), and gocov need the go program which is in $(shell dirname $(GO))